calcom/cal.com
# fix: padding in booking page

## What does this PR do?

Made padding to 24 px all around on booking page

Fixes #9634 

![Screenshot 2023-06-19 205052](https://github.com/calcom/cal.com/assets/20976813/a66291d7-d0af-49d9-9644-8e68c1e60902)


## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Try to book a new event. In the booking page, check if margins are even all around (24px) in the booking form.

## Mandatory Tasks

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
